.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: bash
+sudo: required
+os:
```

Comment:
This isn't needed. Travis doesn't check on anything but Linux unless requested otherwise.